doc/usage/builders/index.rst

```diff
@@ -453,9 +453,9 @@ name is ``rinoh``. Refer to the `rinohtype manual`_ for details.
 .. class:: ChangesBuilder
 
    This builder produces an HTML overview of all :rst:dir:`versionadded`,
-   :rst:dir:`versionchanged` and :rst:dir:`deprecated` directives for the
-   current :confval:`version`.  This is useful to generate a ChangeLog file, for
-   example.
+   :rst:dir:`versionchanged`, :rst:dir:`deprecated` and :rst:dir:`versionremoved`
+   directives for the current :confval:`version`.  This is useful to generate a
+   ChangeLog file, for example.
```

Comment:
```suggestion
   changelog file, for example.
```

FWIW, pretty much everywhere else, e.g. [Wikipedia](https://en.wikipedia.org/wiki/Changelog), [Keep a changelog](https://keepachangelog.com/en/1.1.0/), and in practical use I've seen this spelled "changelog" (or "change log"), rather than the awkward CaptialCamelCase "ChangeLog".

Reply:
Note:

> the [GNU](https://en.wikipedia.org/wiki/GNU) ([Automake](https://en.wikipedia.org/wiki/Automake)) canonical naming convention for the file is ChangeLog

I expect that this is the provenance of the capitalisation.

Reply:
Ah, gotcha—I knew I'd seen that _somewhere_. (Of course, the GNU coding standards calls their "LICENSE" file "COPYING", unlike most other places nowadays, and their `CONTRIBUTING` file `HACKING`, as well as implying that no file extension should be added, which significantly inconveniences cross-platform usage and modern automated tooling).

Reply:
(Also FWIW, Keep a changelog's [FAQ](https://keepachangelog.com/en/1.1.0/#frequently-asked-questions) has an entry regarding file naming:)

> ### What should the changelog file be named?
> Call it `CHANGELOG.md`. Some projects use `HISTORY`, `NEWS` or `RELEASES`.
>
> While it's easy to think that the name of your changelog file doesn't matter that much, why make it harder for your end users to consistently find notable changes?